subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/ArtifactsTestListener.kt

```diff
@@ -84,3 +92,5 @@ internal class ArtifactsTestListener(
         }
     }
 }
+
+private const val LOGCAT_LAST_LINES = 300
```

Comment:
Why 300? Could we send all lines?

Reply:
all lines count is unknown, and could be a big file; in fact:

```
 adb logcat -g
main: ring buffer is 2 MiB (1 MiB consumed), max entry is 5120 B, max payload is 4068 B
system: ring buffer is 2 MiB (307 KiB consumed), max entry is 5120 B, max payload is 4068 B
crash: ring buffer is 2 MiB (0 B consumed), max entry is 5120 B, max payload is 4068 B
```

we could alter this by `adb logcat -G <SIZE>` command